[Finish] Update zh_CN.json localizations. 简体中文

batvbs opened the pull request:
> 如果您有对于翻译的建议，请留言。[@dtlnor](https://github.com/dtlnor) [@bgluminous](https://github.com/bgluminous)

### 需要改进的翻译清单：

1. 部分内容不符合实际功能表现。✅

| 英文               | 直译     | 对应功能                | 备注                                              |
| ------------------ | -------------- | ----------------------- | ------------------------------------------------- |
| favorites          | 收藏夹   | 收藏夹(已保存)                  | 在txt2img页面和设置均为save保存，统一语义 |
| CFG Scale          | CFG 规模 | 提示词相关性(CFG Scale) |                                                   |
| Denoising strength | 去噪强度 | 重绘幅度(Denoising strength) |  |
| inpaint | 去瑕疵 | 局部重绘 |  |

| inpaint | |
|----|----|
| 去瑕疵 | 百度翻译 |
| 填补，修复，修补 | 有道翻译 |
| 修补 | 谷歌翻译|
| 绘画，涂料，上漆 | Deepl翻译 |
| 绘画，去水印，局部重绘 | 用户翻译 |

2. 部分文本为自造词，或为专业词汇，用户可能难以理解。

| 英文     | 旧       | 新       | 备注                                         |
| -------- | -------- | -------- | -------------------------------------------- |
| Extras   | 后处理   | 更多     | ✅原义 额外                                   |
| grids    | 概览图   | 宫格图   | ✅原义 网格                                   |
| forever  | 不停的   | 无限     | ✅                                            |

3. 部分内容没有被翻译✅

[35c45df](https://github.com/batvbs/stable-diffusion-webui-localizations/commit/35c45df28b303a05d56a13cb56d4046f08cf8c25)


4. 部分内容有更大众的翻译（其他软件普遍采用）✅

| 英文            | 旧         | 新   | 备注 |
| --------------- | ---------- | ---- | ---- |
| Just resize     | 只缩放     | 拉伸 |      |
| Crop and resize | 缩放并剪裁 | 裁剪 |      |
| Resize and fill | 缩放并填充 | 填充 |      |

batvbs committed:
更新 zh_CN.json

dtlnor commented:
1. 部分文本为繁体字，或为台湾地区的地方词汇。
感谢纠正

2. 部分文本为自造词，或偏移英文原义，或为专业词汇，部分用户可能难以理解。
英文 | 旧 | 新 | 我个人的意见
-- | -- | -- | --
Extras | 后处理 | 更多 | 功能符合描述，这里不是文学作品的翻译。这是一件工具的翻译。当然，翻译为_更多_确实无法指责。但个人倾向于“后处理”能够更精准地指出其功用。
Style | 模板风格 |   | 同上，模版风格是为了以功能反推描述。如果完全以原意为准，checkpoint需要翻译为检查点，interrogate 要翻译为审问
inpaint | 内补绘制 | 修补 | Inpainting确实是图像修复的意思。但你要考虑这个词的来源是“补上文物图画上缺失空白的地方”，而后由此行为引申出/概括出修复的含义。我们做inpaint并非是在做“修复”或者“还原”。更何况同时有望文生意的outpaint同时存在。现有翻译是取自那位b站up主的翻译，我认为十分贴切。如果你认为有必要把修补的含义加入这个词汇里面。图内修补/图外修补
Mask | 蒙版 | 修改区域 | Mask有两个通用的翻译，一个叫蒙版，一个叫遮罩（遮罩似乎繁中用得更多）。Mask并不是“修改区域”的意思。特别是当你选中“Inpaint not masked”这个选项的时候，修改的区域就是“非修改区域”了
grids | 概览图 | 网格图 | 概览图也是从功用反过来描述的。重申一遍这不是文学作品，而是工具说明。在保留原有的专业概念下尽可能翻译其功用更合适。

MengNianxiaoyao commented:
style我个人觉得可以翻译成预设，但这样和英文原意不符。grids翻译成缩略图也行吧，grids是生成的图片的集合显示图，缩略图会更恰当点

dtlnor committed:
Update zh_CN.json

update translation content to 35c45df28b303a05d56a13cb56d4046f08cf8c25

dtlnor commented:
> style我个人觉得可以翻译成预设，但这样和英文原意不符。grids翻译成缩略图也行吧，grids是生成的图片的集合显示图，缩略图会更恰当点

确实，预设更符合实际使用情况。但缩略图一般指单个图片的缩略图（Thumbnail），可能不太合适。我个人认为如果真的要完整表达grids的意思。另一位翻译者提到的宫格图可能更合适。~~网格图有种说不出来的怪的感觉~~

MengNianxiaoyao commented:
> 宫格图这个翻译确实更好，网格图没那个味，意思也没那么贴切

MengNianxiaoyao commented:
Upscaler这个我是翻译成超分算法的，也不知道应不应该这么翻译，但这个词的出现的部分地方翻译成超分算法会更好

MengNianxiaoyao commented:
> > style我个人觉得可以翻译成预设，但这样和英文原意不符。grids翻译成缩略图也行吧，grids是生成的图片的集合显示图，缩略图会更恰当点
> 
> 确实，预设更符合实际使用情况。但缩略图一般指单个图片的缩略图（Thumbnail），可能不太合适。我个人认为如果真的要完整表达grids的意思。另一位翻译者提到的宫格图可能更合适。~网格图有种说不出来的怪的感觉~

宫格图这个翻译确实更好，网格图没那个味，意思也没那么贴切

lockmatrix commented:
> Upscaler这个我是翻译成超分算法的，也不知道应不应该这么翻译，但这个词的出现的部分地方翻译成超分算法会更好

不建议缩写，可以考虑用【超分辨率】

要是想更直白可以考虑【提升分辨率】，但我觉得没必要了

MengNianxiaoyao commented:
> > Upscaler这个我是翻译成超分算法的，也不知道应不应该这么翻译，但这个词的出现的部分地方翻译成超分算法会更好
> 
> 不建议缩写，可以考虑用【超分辨率】
> 
> 要是想更直白可以考虑【提升分辨率】，但我觉得没必要了

也是，【超分算法】可能有部分人看不懂，【超分辨率】应该就差不多了，【提升分辨率】没必要，【超分辨率】更专业意思也差不多

batvbs committed:
Merge pull request #2 from dtlnor/add-updated-content

Add new translated content

dtlnor commented:
> 
> 也是，【超分算法】可能有部分人看不懂，【超分辨率】应该就差不多了，【提升分辨率】没必要，【超分辨率】更专业意思也差不多

upscaler里面有传统算法lanczos。lanczos是完全由算法组成的重采样算法。超分显然是不合适的。

batvbs committed:
更新 zh_CN.json

dtlnor commented on localizations/zh_CN.json:
```diff
@@ -500,14 +500,14 @@
+    "fill it with colors of the image": "用图像的颜色(模糊/马赛克)填充它",
+    "keep whatever was there originally": "保留原来的图像，不进行预处理",
+    "fill it with latent space noise": "用潜空间的噪声(随机彩色噪点)填充它",
+    "fill it with latent space zeroes": "用潜空间的零(灰色)填充它",
```
同样，latent space里面的zero不等于“灰色”（rgb数值中也不等于灰色）。

batvbs committed:
Update zh_CN.json

batvbs committed:
长文本添加逗号

batvbs committed:
更新 zh_CN.json

batvbs commented:
没有人觉得“内补绘制”这个自造词很怪吗？

MengNianxiaoyao commented:
> 没有人觉得“内补绘制”这个自造词很怪吗？

但翻译更怪，“上漆”。。。

MengNianxiaoyao commented:
每个选项卡的英文都用括号保留吧

batvbs commented:
> 每个选项卡的英文都用括号保留吧

没有必要。懂英语的用户不需要翻译，不懂英语的用户不需要中英对照。

dtlnor committed:
re-order content to match the dump

Deprecated content was moved to the bottom to keep (somehow) backwards capability.

dtlnor committed:
update new content

dtlnor committed:
Update zh_CN.json

putting back the sign

batvbs commented:
Inpaint翻译为 局部重绘，如何？

batvbs committed:
更新 zh_CN.json

batvbs committed:
Denoising strength

dtlnor commented:
看来 review comment 对你我来说并不是一个很好的沟通渠道。那我一个个说。
1. 如果你认为 [pr#3](https://github.com/batvbs/stable-diffusion-webui-localizations/pull/3) 没有问题的话，请合并，如有，请提出。
2. 关于修改项目
- 生成美术风格 - 为何删除 embedding 关键词。他确实是生成了一个 embedding 文件（但不同于普通的embedding）
- 随机种子(seed) - 我不是很明白加上去的原因。随机种子需要被再次对照seed这个英文才能明白这是什么吗？
- wiki 也在 https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1288559248 提到过了。何况里面的文档是描述/解释。而不是“帮助”
- favorites - favorites是放在logs文件夹里面的图片。而其他tab里面的也依然是“已保存”的文件（output里面）。这里并不适合命名为已保存。且 favorites 已在 https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1287923525 以及 https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1288070570 回应过
- Collect - 已被更新内容取代（在pr#3里）
- fill it with colors of the image - 这里说过了是对原图片进行多次不同半径的高斯模糊达到获取图片平均颜色的效果。模糊是模糊，马赛克是马赛克。请把马赛克去掉。以及，约等于符号（≈）在小号文字上非常容易被错认为等于符号（=）。建议换一种表达方式
- fill it with latent space noise / zeroes - 图像上的噪声确实表现为随机彩色噪点。但 latent space 中填入噪声不等于你喂了一张充满rgb噪声的图片。stable diffusion 里面的 latent space 虽然与输入数据有相同的维度（其他模型中可以是更高维的数据），但不等于是图片或者说rgb数据。在latent space里面填零同理。噪声(≈随机彩色噪点)，零(≈灰色)也是错误的。latent space并不是“图片”。你不可以用图像处理的概念去形容他。Leave it as is.
3. 关于 inpaint
inpaint跟outpaint是对应的一组功能。在翻译的时候要考虑清楚两者的关系。不要把一个如此形象的词想得那么复杂。inpaint往里边儿画。outpaint往外面儿画。https://en.wikipedia.org/wiki/Inpainting 
> Inpainting is a conservation process where damaged, deteriorated, or missing parts of an artwork are **filled** in to present a complete image.

dtlnor committed:
Update zh_CN.json

update tag complete.

batvbs committed:
Merge pull request #3 from dtlnor/add-updated-content-2

Add updated content 2

batvbs commented:
> 看来 review comment 对你我来说并不是一个很好的沟通渠道。那我一个个说。
> 
> 1. 如果你认为 [pr#3](https://github.com/batvbs/stable-diffusion-webui-localizations/pull/3) 没有问题的话，请合并，如有，请提出。

✅已合并。如有合并请求，请在此回复，我能收到邮件。

<br></br>
 

> 2. 关于修改项目
> 
> * 生成美术风格 - 为何删除 embedding 关键词。他确实是生成了一个 embedding 文件（但不同于普通的embedding）

⚪️我认为tab标签不宜太长。
需要此功能的用户点进去也能看到embedding，并不影响对于“确实是生成了一个 embedding 文件”的理解。
 #【这里有一张图片，你能看到吗？】
![image](https://user-images.githubusercontent.com/60730393/198967440-e527da83-3101-4930-8b77-594f14df0785.png)

如果你坚持添加embedding，那我就改回去
<br></br>

> * 随机种子(seed) - 我不是很明白加上去的原因。随机种子需要被再次对照seed这个英文才能明白这是什么吗？

⚪️生成的图片默认文件名里有seed，便于用户对照理解。
<br></br>

> * wiki 也在 [[Finish] Add support for simplified chinese 简体中文化 #3348 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1288559248) 提到过了。何况里面的文档是描述/解释。而不是“帮助”

⚪️改成[Wiki(可供多人协同创作的描述文档)]？
或[Wiki文档]？
<br></br>

> * favorites - favorites是放在logs文件夹里面的图片。而其他tab里面的也依然是“已保存”的文件（output里面）。这里并不适合命名为已保存。且 favorites 已在 [[Finish] Add support for simplified chinese 简体中文化 #3348 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1287923525) 以及 [[Finish] Add support for simplified chinese 简体中文化 #3348 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1288070570) 回应过
> * Collect - 已被更新内容取代（在pr#3里

⚪️output/txt2img文件夹是图像**输出**的历史记录，并不是“已保存”；
log/images文件夹直译是图片日志，实际为保存图片的位置(可在设置里查看，如图所示)
需要指出的是，图像浏览器是他人贡献的扩展，且已从主分支移除。我认为翻译应与主分支统一。
或者，改成 `收藏夹(已保存)`？
 #【这里有一张图片，你能看到吗？】
![image](https://user-images.githubusercontent.com/60730393/198874585-693bc27f-a8be-4e33-b60b-8778bc869d13.png)
<br></br>


> * fill it with colors of the image - 这里说过了是对原图片进行多次不同半径的高斯模糊达到获取图片平均颜色的效果。模糊是模糊，马赛克是马赛克。请把马赛克去掉。以及，约等于符号（≈）在小号文字上非常容易被错认为等于符号（=）。建议换一种表达方式

✅好的，去掉马赛克。
<br></br>

> * fill it with latent space noise / zeroes - 图像上的噪声确实表现为随机彩色噪点。但 latent space 中填入噪声不等于你喂了一张充满rgb噪声的图片。stable diffusion 里面的 latent space 虽然与输入数据有相同的维度（其他模型中可以是更高维的数据），但不等于是图片或者说rgb数据。在latent space里面填零同理。噪声(≈随机彩色噪点)，零(≈灰色)也是错误的。latent space并不是“图片”。你不可以用图像处理的概念去形容他。Leave it as is.

⚪️(图像表现为随机彩色噪点)、(图像表现为无色纯灰)如何？
<br></br>

> 3. 关于 inpaint
>    inpaint跟outpaint是对应的一组功能。在翻译的时候要考虑清楚两者的关系。不要把一个如此形象的词想得那么复杂。inpaint往里边儿画。outpaint往外面儿画。https://en.wikipedia.org/wiki/Inpainting
> 
> > Inpainting is a conservation process where damaged, deteriorated, or missing parts of an artwork are **filled** in to present a complete image.

⚪️争议不是采取何种翻译，而是“内补绘画”这个语义不明的自造词(谷歌搜不到)，绝大多数人无法理解和接受。
我同意修改为任何非自造词，如上诉wiki百科对应的中文词条“图像修复”
<br></br>

batvbs committed:
修改争议部分

batvbs committed:
outpainting

batvbs committed:
将无法本地化的内容移到底部

batvbs committed:
inpaint

batvbs committed:
更新 zh_CN.json

batvbs committed:
bug修复

文本太长，遮挡提示框

MengNianxiaoyao commented:
inpaint翻译为内部修补，outpaint翻译为外部绘制，这样行不行？

batvbs committed:
更新 zh_CN.json

batvbs commented:
> inpaint翻译为内部修补，outpaint翻译为外部绘制，这样行不行？

我认为不应该翻译in，不需要把词拆分，这个词

> inpaint翻译为内部修补，outpaint翻译为外部绘制，这样行不行？

局部修补，向外绘制

batvbs committed:
Update zh_CN.json

dtlnor commented:
> 我认为tab标签不宜太长。
确实（~~其实extension越来越多以后肯定会变成两行的~~）

> 生成的图片默认文件名里有seed，便于用户对照理解。
赞同，不过如果是这样的话，应该把所有会在PNG info里面出现的关键词都标上英文（比如在eta noise seed delta部分标上缩写（ENSD），以及采样器等名词标上sampler）

> 改成[Wiki文档]？
那就文档吧（）

> log/images文件夹直译是图片日志，实际为保存图片的位置(可在设置里查看，如图所示)
~~（你这个直译叫我怎么吐槽）~~ 这点我非常清楚。但强制将save跟favorites（这跟是不是别人的extension还是主分支的零件没有关联）统一在我看来很奇怪。如果要显性地指出收藏夹实际上完全等于保存按钮的默认输出目录，`收藏夹(已保存)`会比较好（虽然我个人认为冗长了点）

> 好的，去掉马赛克，改成高斯模糊
小建议：改成高强度模糊。如果你有ps可以试试模拟这个操作：半径256像素的高斯模糊（mask边缘会被稍低半径高斯模糊结果取代，分别是64、4、2、以及整整好是边缘的地方=无模糊。））。基本上就算是1024大小的图片，半径256像素的高斯模糊也只能看到一种颜色了。

> (图像维度表现为随机彩色噪点)、(图像维度表现为无色纯灰)如何？
noise不等于随机彩色噪点。noise是不包含信息的数值（也就是随机数值）的意思。图片上的噪声会表现为噪点（每个像素的rgb值都是随机出来的）。在声音上会表现会滋滋滋的杂讯。在latent space则是无法表现。latent space的数值需要被decode才可以输出为图像，因此你不能用图像的概念去描述他。所以不要描述他（你加了就画蛇添足了的意思，原文反正也没加描述）。
打个比喻就是，你把图片压缩为rar文件，然后把其中一个区域填了随机数值或者填了零，这个跟“随机彩色噪点”或者“纯灰”是毫无关联的。你只有在解压的时候才能得到图像结果。

> “内补绘画”这个语义不明的自造词
无论是stable diffusion的inpaint或者是webui的outpaint都是此领域独有的概念。你为什么要搜索他？他就是单独的概念，此处的inpaint跟用于油画修复的inpaint也并非同一个行为。我难以同意执着于在中文里面找一个已有的东西去描述一个新的独有的概念（目前你的观点难以说服我）。内部绘制/修复为何不是自造词。外部绘制为什么就不是制造词。执着于inpaint的“修补”而忽视实际的功能同样会让用户不知所谓。
“我认为绝大多数英语用户自己都不认识”，但不认识不代表不能望文生义、不能直接get到对吧。无论是你在文件格式后面加说明（比如PNG图像格式）或者尝试在翻译里面加补充说明（比如上面的）都似乎把用户当成傻子了。但我认为不需要对用户如此无微不至地照顾。看不懂什么是png/jpg百度一下就学好了（就当帮用户巩固基础常识好了）。看到inpaint/outpaint也不明白在干嘛的用户也是试一试就清楚了。虽然不太懂“自造词汇”的定义，但能够望文生义就足够了。
> 即使非要拆分，in此处的含义是“在”，而不是“内”。
inside / outside，come in / get out，indoor / outdoor

**Anyway**，局部重绘、向外绘制也很清楚表达了其功用，我不在这点纠缠了。

dtlnor commented on localizations/zh_CN.json:
```diff
@@ -448,41 +477,76 @@
+    "Number of grids in each row": "每行显示多少格",
+    "Start drawing": "开始绘制",
+    "how fast should the training go. Low values will take longer to train, high values may fail to converge (not generate accurate results) and/or may break the embedding (This has happened if you see Loss: nan in the training info textbox. If this happens, you need to manually restore your embedding from an older not-broken backup).\n\nYou can set a single numeric value, or multiple learning ra ...
+    "Separate prompts into parts using vertical pipe character (|) and the script will create a picture for every combination of them (except for the first part, which will be present in all combinations)": "用垂直的管道字符 | 将提示语分成若干部分，脚本将为它们的每一个组合创建一幅图片（除了第一部分，所有的组合都会出现）",
```
```suggestion
    "Separate prompts into parts using vertical pipe character (|) and the script will create a picture for every combination of them (except for the first part, which will be present in all combinations)": "用竖线分隔符(|)将提示词分成若干部分，脚本将为它们的每一个组合创建一幅图片（除了被分割的第一部分，所有的组合都会包含这部分）",
```

dtlnor commented on localizations/zh_CN.json:
```diff
@@ -109,53 +108,52 @@
     "Image for img2img": "图生图的图像",
-    "Image for inpainting with mask": "用于内补绘制蒙版内容的图像",
+    "Drop Image Here": "拖拽图像到此",
+    "Image for inpainting with mask": "用于提取蒙版的图像",
```
看了看代码，这部分是不显示的。但还是改一下好。实际上是Draw Mask里面手动画mask的那个框的label
```suggestion
    "Image for inpainting with mask": "用于局部重绘并手动画蒙版的图像",
```

dtlnor commented on localizations/zh_CN.json:
```diff
@@ -109,53 +108,52 @@
-    "Use an empty output directory to save pictures normally instead of writing to the output directory.": "指定一个空的文件夹为输出目录而非默认的 output 文件夹为输出目录",
-    "Disabled when launched with --hide-ui-dir-config.": "启动 --hide-ui-dir-config 时禁用",
+    "Inpaint at full resolution": "以完整分辨率进行局部重绘",
+    "Inpaint at full resolution padding, pixels": "填补像素",
```
这个选项只在启用“以完整分辨率进行内补绘制”时生效
```suggestion
    "Inpaint at full resolution padding, pixels": "以完整分辨率进行内补绘制 - 填补像素",
```

batvbs committed:
Update zh_CN.json

batvbs committed:
Update localizations/zh_CN.json

Co-authored-by: dtlnor <dtlnor@hotmail.com>

batvbs committed:
Update localizations/zh_CN.json

Co-authored-by: dtlnor <dtlnor@hotmail.com>

dtlnor commented:
想问一下以`完整分辨率进行局部重绘 - 填补像素`是有什么说法吗？

MengNianxiaoyao commented:
> 想问一下以`完整分辨率进行局部重绘 - 填补像素`是有什么说法吗？

额，这里我是按照字面翻译，不知道其他人是怎么想的

MengNianxiaoyao commented:
我觉得可以参考一下其他语言的翻译文本，然后对照翻译一下，其他语言机翻过来就行

liggest commented:
原来开了新的 PR，好耶

liggest commented on localizations/zh_CN.json:
```diff
@@ -181,7 +179,7 @@
     "Will upscale the image to twice the dimensions; use width and height sliders to set tile size": "将图像放大到两倍尺寸； 使用宽度和高度滑块设置图块尺寸(tile size)",
     "Tile overlap": "图块重叠的像素(Tile overlap)",
-    "Upscaler": "放大算法",
+    "Upscaler": "Upscaler",
```
```suggestion
```
与下面重复了

liggest reviewed (commented):
```json
  "Open for Clip Aesthetic!": "打开美术风格 Clip!",
```
暂时换成 `打开以调整美术风格！` 会不会更容易理解一点，虽然我也不知道 Clip 怎么翻译…

```json
  "Slerp interpolation": "Slerp 插值",
  "Slerp angle": "Slerp 角度",
```
[维基](https://zh.wikipedia.org/wiki/%E7%90%83%E9%9D%A2%E7%B7%9A%E6%80%A7%E6%8F%92%E5%80%BC)提供了 `球面线性插值` 的翻译，不太懂，不知道这里是不是那个意思

```json
  "uniform": "均勻",
```
`均匀`
```json
  "Hypernetwork": "超网络(Hypernetwork)"
```
只有这里是 `超网络`，如果决定用这个译名，应该还需要统一文件里的其它地方

```json
  "Single Image": "单个图像",
```
无关紧要的地方，可能 `单张图像` 更顺耳一点？

liggest commented:
上面好像提到了 outpainting 和扩展有关，叫 `扩展绘制` 或者 `拓展绘制` 如何？

----

关于 conditioning mask

```json
  "Cond. Image Mask Weight": "自适应图像蒙版强度",
  "Inpainting conditioning mask strength": "局部重绘的自适应蒙版强度",
  "Only applies to inpainting models. Determines how strongly to mask off the original image for inpainting and img2img. 1.0 means fully masked, which is the default behaviour. 0.0 means a fully unmasked conditioning. Lower values will help preserve the overall composition of the image, but will struggle with large changes.": "仅适用于局部重绘专用的模型。 决定了蒙版在局部重绘以及图生图中屏蔽原图内容的强度。 1.0 表示完全屏蔽，这是默认行为。 0.0 表示完全不屏蔽。 较低的值将有助于保持图像的整体构图，但很难遇到较大的变化。",
```

我不太了解这个功能，请问 conditioning 为啥翻译成自适应？看注解好像也没觉得和这个意思比较接近

batvbs committed:
Update localizations/zh_CN.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor commented:
> ```json
>   "Open for Clip Aesthetic!": "打开美术风格 Clip!",
> ```
> 
> 暂时换成 `打开以调整美术风格！` 会不会更容易理解一点，虽然我也不知道 Clip 怎么翻译…

 `打开以调整 Clip 的美术风格！`

> ```json
>   "Slerp interpolation": "Slerp 插值",
>   "Slerp angle": "Slerp 角度",
> ```
> 
> [维基](https://zh.wikipedia.org/wiki/%E7%90%83%E9%9D%A2%E7%B7%9A%E6%80%A7%E6%8F%92%E5%80%BC)提供了 `球面线性插值` 的翻译，不太懂，不知道这里是不是那个意思

是，只是翻译成中文估计懂的人也不多我就没翻了（嫌长），替换成`球面线性插值`其实没有问题
> 关于 conditioning mask

看了看代码跟原理，是我翻错了。完全不是这个意思。我稍后提交修改

dtlnor committed:
Update zh_CN.json

include new changes

dtlnor committed:
re-order json content

batvbs committed:
Update zh_CN.json

batvbs committed:
Update zh_CN.json

batvbs commented:
> 想问一下以是有什么说法吗？`完整分辨率进行局部重绘 - 填补像素`

Inpaint at full resolution
Normally, inpainting resizes the image to the target resolution specified in the UI. With Inpaint at full resolution enabled, only the masked region is resized, and after processing it is pasted back to the original picture. This allows you to work with large pictures and allows you to render the inpainted object at a much larger resolution.

dtlnor commented:
> > 想问一下以是有什么说法吗？`完整分辨率进行局部重绘 - 填补像素`
> 
> Inpaint at full resolution Normally, inpainting resizes the image to the target resolution specified in the UI. With Inpaint at full resolution enabled, only the masked region is resized, and after processing it is pasted back to the original picture. This allows you to work with large pictures and allows you to render the inpainted object at a much larger resolution.

What do you mean by posting the explanation about `Inpaint at full resolution`? This is unrelated to the padding of `Inpaint at full resolution`.
`Inpaint at full resolution padding` are enabled ONLY when `Inpaint at full resolution` get enabled.

liggest commented:
> 上面好像提到了 outpainting 和扩展有关，叫 `扩展绘制` 或者 `拓展绘制` 如何？

如果要和 `局部重绘` 对仗，或许 `外部扩绘` ?

batvbs commented:
>


> > 上面好像提到了 outpaint 和扩展有关，叫 或者 如何？`扩展绘制``拓展绘制`
> 
> 如果要和 对仗，或许 ?`局部重绘``外部扩绘`
> 
> 虽然这样又有点自造词的嫌疑了 (;¬_¬)

向外绘制已经清晰的表达了其作用。

dtlnor commented:
So, may I ask if you are unwilling to discuss this with me anymore? If so, I wouldn't force it. The update has been on hold for a long time, and I will not keep working on your branch.

batvbs commented:
> So, may I ask if you are unwilling to discuss this with me anymore? If so, I wouldn't force it. The update has been on hold for a long time, and I will not keep working on your branch.

你不是说“我稍后提交修改”吗？我一直在等你啊。
没有没有什么需要修改的，准备提交合并请求。

batvbs committed:
Inpaint at full resolution

dtlnor commented:
> > So, may I ask if you are unwilling to discuss this with me anymore? If so, I wouldn't force it. The update has been on hold for a long time, and I will not keep working on your branch.
> 
> 你不是说“我稍后提交修改”吗？我一直在等你啊。 如果没有什么需要修改的，准备提交合并请求。 If there is no amendment, Submit the merge request.

從你這裡我得不到足夠反饋。我等你的反饋等了一整天。我難以理解這種拒絕溝通的行為。所以如我所說，如果你拒絕溝通，我不會繼續在這個分支上更新。我的修改會直接提交而不是經由你的分支。

batvbs commented:
> 從你這裡我得不到足夠反饋。我等你的反饋等了一整天。我難以理解這種拒絕溝通的行為。所以如我所說，如果你拒絕溝通，我不會繼續在這個分支上更新。我的修改會直接提交而不是經由你的分支。

我对你的每项review comment都进行了回复。根据你的态度，或许此回复“仅自己可见”？如果是这样，我表示抱歉，对不起。

这里有一张图片
![IMG_20221103_084200](https://user-images.githubusercontent.com/60730393/199627955-edd29481-ad7e-4f3c-8e81-d937f32ca260.jpg)

bgluminous commented on localizations/zh_CN.json:
```diff
@@ -59,12 +58,12 @@
-    "Denoising strength": "去噪强度",
-    "Batch count": "批次",
-    "Batch size": "批量",
+    "Denoising strength": "重绘幅度(Denoising strength)",
```
我觉得 “去噪强度” 更加合适

batvbs replied:
> 我觉得 “去噪强度” 更加合适
CFG Scale	CFG 规模 	提示词相关性(CFG Scale)
？
我觉得部分内容可以采取实际功能翻译，而非直译。

dtlnor replied:
Denoising strength其实也是术语，特指在SD这里。这里的denoise其实并非常规的denoise（不是那种后处理滤镜）。而是同时指“加噪”强度（所以我一开始的翻译“模糊”是错的，那种只适合描述常规降噪滤镜）。在sd进行sampling的过程中，他要先有noise（高斯噪点）然后逐渐从noise“恢复”成图片。这个“恢复”的过程就叫denoise。而假设我们有1份noise（100%），step则决定每个step去除多少%的noise。比如100步就是每步1%。
i2i的生成过程就是，假如你的denoise强度是1.0，那他会加100%的噪点，然后跑100%的sampling。而假如denoise是0.0，那他就不加噪点，也就是不做sampling，直接encode进latent然后再直接decode回来。

bgluminous commented:
@dtlnor 有什么可用的联系方式吗）

bgluminous commented:
@dtlnor 有什么可用的联系方式吗）

dtlnor commented:
> > 從你這裡我得不到足夠反饋。我等你的反饋等了一整天。我難以理解這種拒絕溝通的行為。所以如我所說，如果你拒絕溝通，我不會繼續在這個分支上更新。我的修改會直接提交而不是經由你的分支。
> 
> 我对你的每项review comment都进行了回复。根据你的态度，或许此回复“仅自己可见”？如果是这样，我表示抱歉，对不起。
> 
> 这里有一张图片 ![IMG_20221103_084200](https://user-images.githubusercontent.com/60730393/199627955-edd29481-ad7e-4f3c-8e81-d937f32ca260.jpg)

明白了，这完全是误会。你的回复只要是pending就只有你可见

bgluminous commented on localizations/zh_CN.json:
```diff
@@ -59,12 +58,12 @@
-    "Seed": "随机种子",
-    "Extra": "额外参数",
+    "Seed": "随机种子(seed)",
+    "Extra": "▼",
```
不推荐这种方式来表达翻译

batvbs replied:
> 不推荐这种方式来表达翻译

此处本地化后只能显示一个全角字符，不知道原因。或许可以修改ui.py解决此问题？

batvbs replied:
此外，旧版本的webui，此处是▼

dtlnor replied:
確實，本地化後這裡是豎排的
額
外
參
數
一言難盡

bgluminous commented on localizations/zh_CN.json:
```diff
@@ -81,16 +80,17 @@
     "Is negative text": "是反向提示词",
     "Script": "脚本",
-    "Embedding to Shareable PNG": "将 Embedding 转换为可分享的 PNG",
+    "Embedding to Shareable PNG": "将 Embedding 转换为可分享的 PNG 图片文件",
```
PNG作为图片格式我觉得不需要再重复说明了

bgluminous commented:
翻译中有一些一致性问题 例如 中英文间没有加入空格分隔

```json
[
    "KaimingUniform": "Kaiming均匀",
    "KaimingNormal": "Kaiming正态",
    "XavierUniform": "Xavier均匀",
    "XavierNormal": "Xavier正态",
]
```

dtlnor commented:
> @dtlnor 有什么可用的联系方式吗）

你是不是用github会比较不方便（抱歉我之前没考虑到这件事情），如果是这样的话，你开一个qq群（你之前开的那个可以用吗？）我会申请加入的

bgluminous commented:
> > @dtlnor 有什么可用的联系方式吗）
> 
> 你是不是用github会比较不方便（抱歉我之前没考虑到这件事情），如果是这样的话，你开一个qq群（你之前开的那个可以用吗？）我会申请加入的

主要是非实时的交流有时确实不方便， 群号码：418316375

batvbs commented:
不好意思，我评论不小心点到了某个按钮，草稿又变成了Open，正在寻找切换回草稿的按钮

dtlnor committed:
Merge branch 'pr/3915' into add-updated-content-3

dtlnor committed:
unify translation style

dtlnor committed:
Update zh_CN.json

- re-order some element
- update new content

dtlnor committed:
polish translation content

dtlnor commented:
https://github.com/batvbs/stable-diffusion-webui-localizations/pull/4
已提交pr, 请查看

dtlnor commented:
> 不好意思，我评论不小心点到了某个按钮，草稿又变成了Open，正在寻找切换回草稿的按钮

open就open, 不碍事 ~~没必要把branch都删了www~~

batvbs committed:
更新 zh_CN.json

batvbs commented:
> [batvbs#4](https://github.com/batvbs/stable-diffusion-webui-localizations/pull/4) 已提交pr, 请查看
> 
> > 翻译中有一些一致性问题 例如 中英文间没有加入空格分隔
> 
> 也在pr里修了

存在合并冲突，无法合并。晚上我试试怎么解决

dtlnor committed:
Merge branch 'pr/3915' into add-updated-content-3

dtlnor committed:
polish translation content

dtlnor commented:
> 存在合并冲突，无法合并。晚上我试试怎么解决

我合并了你最新的分支，应该没有问题了。你看看
（以及对新增的内容修正了一下）

batvbs committed:
Merge pull request #4 from dtlnor/add-updated-content-3

Add updated content 3

batvbs committed:
Update localizations/zh_CN.json

Co-authored-by: dtlnor <dtlnor@hotmail.com>

batvbs committed:
Update localizations/zh_CN.json

Co-authored-by: dtlnor <dtlnor@hotmail.com>

liuqipianxin commented:
有一些按钮或提示并没有被翻译
<img width="675" alt="image" src="https://user-images.githubusercontent.com/34701839/199679196-2b56c186-7982-4658-bbf2-025aedd88b6c.png">
<img width="577" alt="image" src="https://user-images.githubusercontent.com/34701839/199679239-d711c2f6-fb12-47d0-8f60-b62c18c90ba5.png">

bgluminous commented:
不推荐把 [WIKI] 翻译成 [帮助]

dtlnor commented:
> 有一些按钮或提示并没有被翻译 <img alt="image" width="675" src="https://user-images.githubusercontent.com/34701839/199679196-2b56c186-7982-4658-bbf2-025aedd88b6c.png"> <img alt="image" width="577" src="https://user-images.githubusercontent.com/34701839/199679239-d711c2f6-fb12-47d0-8f60-b62c18c90ba5.png">

这些是bug，加了都不能被识别

batvbs committed:
将无法本地化的内容移到底部

batvbs committed:
部分无法本地化内容的替代方案

batvbs commented:
> 有一些按钮或提示并没有被翻译 <img alt="image" width="675" src="https://user-images.githubusercontent.com/34701839/199679196-2b56c186-7982-4658-bbf2-025aedd88b6c.png"> <img alt="image" width="577" src="https://user-images.githubusercontent.com/34701839/199679239-d711c2f6-fb12-47d0-8f60-b62c18c90ba5.png">

这是webui本地化模块的bug，你应该在[这里提交](https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues)

我尝试了一个替代方案
![image](https://user-images.githubusercontent.com/60730393/199720766-2275c728-0e53-4a19-b70e-7a0c64addc8c.png)
![image](https://user-images.githubusercontent.com/60730393/199720842-894db8a7-8db9-43e8-a76a-4ce60d721d1b.png)

batvbs commented:
如果没有其它问题，已准备好提交合并请求。

batvbs commented:
@AUTOMATIC1111

batvbs committed:
更新 zh_CN.json
